src/index.html

```diff
@@ -115,6 +118,54 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
           collaboration, and confidence.</h5>
       </section>
 
+      <section id="reading-challenge" class="bg-pink col-12 py-5">
+        <div class="text-center col-lg-7 col-xs-8 mx-auto mb-3">
+          <a href="https://lincolnlibraries.org/" id="LincolnCityLibraries" target="_blank" rel="noopener">
+            <img class="logo library" src="/assets/img/partners/lincolnlibraries.png" alt="Lincoln City Libraries">
+          </a>
+          <h2 class="font-weight-bold">2021 STEM Reading Challenge</h2>
+          <i>
+            April 1, 2021 - May 15, 2021
+          </i>
+        </div>
+
+        <div class="col-12 d-flex justify-content-center">
+          <div class="col-lg-8 col-xs-10">
+            <h3 class="text-center mb-4">Join Girls Code Lincoln and Lincoln City Libraries for a STEM Reading Challenge!</h3>
+            <p >
+              By taking part in this challenge, children can learn about the rich history and
```

Comment:
This text is a tad small - I'd recommend a little bigger!

Reply:
oh yeah good point - I'll fix that 👍